stages/org.osbuild.chattr

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python3
+"""
+Runs `chattr` to make file/directory immuatble.
+"""
+
+import os
+import sys
+import subprocess
+from typing import Dict
+from urllib.parse import ParseResult, urlparse
+
+import osbuild.api
+
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "required": ["path"],
```

Comment:
In the future we can add more attributes and then we'll need to require OneOf them but for now we only have one so wwe might as well require it to be passed.

```suggestion
  "required": ["path","immutable"],
```